test/uk/gov/hmrc/apiplatformadminapi/mocks/ThirdPartyOrchestratorConnectorMockModule.scala

```diff
@@ -39,4 +41,16 @@ trait ThirdPartyOrchestratorConnectorMockModule extends MockitoSugar with Argume
     def verifyCalledWith(applicationId: ApplicationId) =
       verify(mockThirdPartyOrchestratorConnector).getApplication(eqTo(applicationId))(*)
   }
+
+  object QueryUsers {
```

Comment:
I think these objects are usually named the same as the methods they are mocking, so `GetBySessionId` here